Write docs for new UUID type (#1512)
neo4j/docs-cypher: modules/ROOT/pages/functions/index.adoc

|===

[[header-query-functions-uuid]]
== UUID functions

> UUID functions allow you to create UUID values and return their most/least significant types.

Reviewer: ```suggestion
UUID functions allow you to create xref:values-and-types/uuid.adoc[`UUID` values] and return their most/least significant types.
```

[options="header"]
|===
| Function | Signature | Description

1.1+| xref::functions/uuid.adoc#functions-uuid[`uuid()`]
| `uuid() :: UUID`
| Returns a randomly generated `UUID`. label:cypher[Cypher 25 only] label:new[Introduced in Neo4j 2026.08]

1.1+| xref::functions/uuid.adoc#functions-uuid[`uuid()`]
| `uuid(name :: STRING) :: UUID`

> | The `STRING` to convert to a `UUID`, this should be 32 hexadecimal digits displayed in 5 groups, separated by 4 hyphens. label:cypher[Cypher 25 only] label:new[Introduced in Neo4j 2026.08]

Reviewer: Removed argument description part.
```suggestion
| The `STRING` to convert to a `UUID`.
Converts a `STRING` value to a `UUID`. label:cypher[Cypher 25 only] label:new[Introduced in Neo4j 2026.08]
```

1.1+| xref::functions/uuid.adoc#functions-uuid[`uuid()`]
| `uuid(mostSigBits :: INTEGER, leastSigBits :: INTEGER) :: UUID`

> | Converts the given `INTEGER` values to a `UUID`. label:cypher[Cypher 25 only] label:new[Introduced in Neo4j 2026.08]

Reviewer: ```suggestion
| Converts an `INTEGER` value to a `UUID`. label:cypher[Cypher 25 only] label:new[Introduced in Neo4j 2026.08]
```
